backend/compact-connect/lambdas/python/common/cc_common/data_model/schema/adverse_action/record.py

```
@@ -0,0 +1,78 @@
+# ruff: noqa: N801, N815  invalid-name
+from uuid import uuid4
+
+from marshmallow import ValidationError, pre_dump, validates_schema
+from marshmallow.fields import UUID, Boolean, Date, DateTime, String
```

Ruff (0.8.2)

`marshmallow.fields.Boolean` imported but unused

Remove unused import: `marshmallow.fields.Boolean`

(F401)